index.html

```diff
@@ -162,32 +161,56 @@
             card.appendChild(editButton);
 
             app.appendChild(card);
-        });
-
-        const scoringSection = document.createElement("div");
-        scoringSection.classList.add("scoring");
+        }
 
-        const scoreLabel = document.createElement("label");
-        scoreLabel.textContent = "Score the list:";
-        scoringSection.appendChild(scoreLabel);
+        function createAddButton() {
+            const addButton = document.createElement("button");
+            addButton.classList.add("btn", "waves-effect", "waves-light", "add-button");
+            addButton.innerHTML = '<i class="material-icons">add</i>';
+            addButton.addEventListener("click", () => {
+                const newTimer = {
+                    description: "New Timer",
+                    duration: 60
+                };
+                timers.push(newTimer);
+                createCard(newTimer);
+                updateUrlWithTimers(timers);
+            });
+            app.appendChild(addButton);
+        }
```

Comment:
Add the "Add Timer" button under the card list. Place the following code after the app.appendChild(scoringSection); line in the <script> section: